components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementService.java

```diff
@@ -96,6 +95,22 @@ public interface WorkflowManagementService {
     void addWorkflow(Workflow workflowDTO,
                      List<Parameter> parameterList, int tenantId) throws WorkflowException;
 
+    /**
+     *Get Template ID from workflow Id
+     * @param workflowId workflow ID retrieve from workflow request
+     * @return Template ID
+     * @throws WorkflowException
+     */
+    String getTemplateId(String workflowId) throws  WorkflowException;
+
+    /**
+     * Get external Workflow Identifier (Workflow Deploy ID)
+     * @param workflowId workflow ID retrieve from workflow request
```

Comment:
add a new line after dis and explain the return and all the parameters

Reply:
fix by : https://github.com/wso2/carbon-identity-framework/pull/4519/commits/038a1a1ca8e9021e90440538fcf4251b989aa3e0